README.md

```diff
@@ -209,29 +223,47 @@ You can also select the `Reattach children` checkbox to move along all children
 </details>
 
 <details>
-<summary>Other actions</summary>
+<summary><strong>Override fork point</strong></summary>
```

Comment:
Whoops... that's the first time `fork point` is introduced, would be nice to explain the term as the very first thing in the paragraph